### PR TITLE
notify: adapt to the newest version of processes smart-contract

### PR DESCRIPTION
**Pull request by jordipainan**
Updated vocdoni-node version.

### CODE REVIEW

File: notify/ethereum.go [hunk 1]
```diff
@@ -12,8 +12,8 @@ import (
 )
 
 var ethereumEventList = map[string]string{
-	// NewProcess(bytes32 processId, uint16 namespace)
-	"processesNewProcess": "0x2399440b5a42cbc7ba215c9c176f7cd16b511a8727c1f277635f3fce4649156e",
+	// NewProcess(bytes32 processId, uint32 namespace)
+	"processesNewProcess": "0x3b1cc0fc696cbe654bd83494847cc7890f2ae0e05a79dfbd6c1892061fbf3404",
```
Comment (emmdim) [inline notify/ethereum.go:16]: This contract to which  deployment environment does it refer to? Do one address represent all environments?

**Review by emmdim — APPROVED** [review]

**Comment by emmdim** [comment]
LGTM

File: notify/ethereum.go (hunk 1 above)
Comment (jordipainan) [reply notify/ethereum.go:16]: this is not an address, is the hash of the event

Comment (emmdim) [reply notify/ethereum.go:16]: ooops. Sorry :-)

**Pull request merged**